src/ViewInstance.js

```diff
@@ -932,6 +934,21 @@ class ViewInstance extends React.Component {
           </MenuSection>
         )}
 
+        {isSourceLinkedData && (
+          <MenuSection
+            id="ld-menu-section"
+            label={intl.formatMessage({ id: 'ui-inventory.ld.label' })}
+          >
+            <ActionItem
+              id="edit-resource-in-ld"
+              icon="edit"
+              messageId="ui-inventory.editInLinkedDataEditor"
+              onClickHandler={onToggle}
+              disabled={!canAccessLinkedDataOptions}
```

Comment:
It's a pattern in Folio to hide buttons that can't be clicked due to missing permissions. Can we hide the menu action instead here?

Reply:
The ticket states that the button needs to be present but disabled

Reply:
Yes but requirements can be discussed and refined, and it's good to be consistend with how permissions are applied across other apps

Reply:
I've created a chat including you @BogdanDenis and our PO Doug Loynes so that you can propose this to him directly (I think he has some questions about that, too)

Reply:
Done @BogdanDenis